backend/api/v0/urls.py

```diff
@@ -39,6 +40,10 @@
 
     path('courses/offerings/', views.offerings),
     path('courses/<slug:slug>/', views.course),
+    path('courses/team/join/request/', teams_api.request_to_join_team),
```

Comment:
Does this need to be under courses? Feel like team is just better -> also lets keep the plural so :"teams"

Reply:
Yeah agreed